.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,30 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/python
+{
+	"name": "dbt-dremio-devcontainer",
+	// Use a Docker Compose file
+	"dockerComposeFile": "../docker-compose.yml",
+	"service": "python", 
+	"workspaceFolder": "/usr/src/app",
+
+	"runArgs": [
+  		"--userns=keep-id:uid=1000,gid=1000"
+ 	],
+ 	"containerUser": "vscode",
+	"updateRemoteUserUID": true,
+
+	// Configure tool-specific properties.
+	"customizations": {
+		"vscode": {
```

Comment:
What if a developer doesn't have vscode installed, will it become a blocker for them to use dev containers?

Reply:
No, this is part of the devcontainers standard. We could also add customizations for other editors in the feature (see [here](https://containers.dev/supporting))